lua/autorun/client/cl_parachute.lua

```diff
@@ -101,17 +117,27 @@ function CFC_Parachute.CreateToggleButton( x, y, ind, panel, w, h )
     local convar = convarName and GetConVar( convarName )
     local offVal = buttonData.ConVarOff or "0"
     local onVal = buttonData.ConVarOn or "1"
+    local svVal = buttonData.ConVarServerChoice
     local offText = buttonData.TextOff or "Enable [UNDEFINED]"
     local onText = buttonData.TextOn or "Disable [UNDEFINED]"
+    local hoverText = buttonData.HoverText or ( convar and convar:GetHelpText() )
 
     button.cfcParachuteIsOn = buttonData.IsOn or function()
-        return convar:GetString() == onVal
+        local curVal = convar:GetString()
+
+        return curVal == onVal or ( curVal == svVal and GetConVar( convarName .. "_sv" ):GetString() == onVal )
     end
 
     button.cfcParachuteIntendedText = function()
         return button.cfcParachuteIsOn() and onText or offText
     end
 
+    button.cfcParachuteIntendedHoverText = function()
+        local text = hoverText .. "\n\nCurrently "
```

Comment:
I like to keep newline characters in their own string. Makes things look a little nicer.
```suggestion
        local text = hoverText .. "\n\n" .. "Currently "
```